tests/test_factory.py

```diff
@@ -466,6 +466,15 @@ def test_random_pyfloat(self):
         with pytest.raises(ValueError):
             provider.pyfloat(left_digits=0, right_digits=0)
 
+    def test_pyfloat_in_range(self):
+        from faker.providers.python import Provider
+        provider = Provider(self.generator)
+
+        for _ in range(99):
```

Comment:
Let's find a seed value that would've failed the test before the fix an use that, instead of iterating 99 times. See ` test_negative_pyfloat` below for an example

Reply:
Good call!  A seed of 5 works, pushing fix now.